docs/src/main/asciidoc/security-authorize-web-endpoints-reference.adoc

```diff
@@ -4,7 +4,7 @@ and pull requests should be submitted there:
 https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
 ////
 [id="security-authorize-web-endpoints-reference"]
-= Authorize web endpoints
+= Authorization of web endpoints
```

Comment:
```suggestion
= Authorizing web endpoints
```

Reply:
The above heading format is good regarding the Diataxes reference format we are using in this repo. But thank you so much for the rest of the suggestions! :) Will add for sure!